PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCore/NativeAssets/NativeAdConfiguration.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  NativeAdConfiguration.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+public class NativeAdConfiguration: NSObject, NSCopying {
+
+    /// Version of the Native Markup version in use.
+    @objc public var version: String? {
+        get { markupRequestObject.version }
+        set { markupRequestObject.version = newValue }
+    }
+    
+    /// [Recommended]
+    /// [Integer]
+    /// The context in which the ad appears.
+    @objc public var context: PBMNativeContextType {
+        get {
+            PBMNativeContextType(rawValue: (markupRequestObject.context?.intValue) ??
+                                    PBMNativeContextType.undefined.rawValue) ?? .undefined
+        }
+        set {
+            markupRequestObject.context = newValue == .undefined ? nil : NSNumber(value: newValue.rawValue)
+        }
+    }
+
+    /// [Integer]
+    /// A more detailed context in which the ad appears.
+    @objc public var contextsubtype: PBMNativeContextSubtype {
+        get {
+            PBMNativeContextSubtype(rawValue: (markupRequestObject.contextsubtype?.intValue) ??
+                                        PBMNativeContextSubtype.undefined.rawValue) ?? .undefined
```

Comment:
Shouldn't we return here the actual value of markupRequestObject. contextsubtype ?